get_bullshtml.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -uex
+
+bullshtml_vers='master'
+bullshtml_url='https://github.com/simplivity/bullshtml/archive/'
+bullshtml_dnl="${bullshtml_vers}.tar.gz"
+bullshtml_tar="bullshtml-${bullshtml_vers}.tar.gz"
+
+if [ -e "${bullshtml_tar}" ]; then
+  zflag=" -z ${bullshtml_tar}"
+else
+  zflag=
+fi
```

Comment:
You don't need any of this.  The `-z` flag works with `curl` even if the file doesn't exist.  In such a case, it simply doesn't send an `If-Modified-Since:` header and fetches unconditionally.